module/VuFind/src/VuFind/Db/Table/ResourceTags.php

```diff
@@ -706,7 +706,7 @@ public function getResourceTags(
         $select->join(
             ['r' => 'resource'],
             'resource_tags.resource_id = r.id',
-            ['title' => 'title']
+            ['title' => 'title', 'record_id', 'source']
```

Comment:
Do we really need the `'title' => 'title'` syntax here, or can we simplify to just `'title'`?